components/Team/Team.tsx

```diff
@@ -0,0 +1,59 @@
+import { gql } from '@apollo/client';
+import { useTeamQuery } from '../../src/graphql/types';
+import { useIndexQuery } from '../../src/graphql/types';
+
+interface Props {
+  team_id: string;
+}
+
+gql`
+  query Team($team_id: String!) {
+    Team(team_id: $team_id) {
+      name
+    }
+  }
+`;
+gql`
+  query Index {
+    allTeams {
+      team_id
+      name
+      managers {
+        nickname
+        manager_id
+      }
+      team_logos {
+        url
+      }
+    }
+  }
+`;
+
+const Teams = () => {
+  const { data, loading } = useIndexQuery();
+  console.log(data);
+  const teamList = data?.allTeams.map((team, i) =>
+    team.team_logos.map((logo, l) =>
+      team.managers.map((manager, m) => (
+        <li key={manager?.nickname}>
+          <div>Team Name: {team.name}</div>
+          <div>Manager: {manager?.nickname}</div>
+          <div>Logo:</div>
+          <img src={logo?.url} alt='team logo' />
+        </li>
+      ))
```

Comment:
One thing I noticed is that for our team where you're the co-manager, you're showing up too. Is there a way we can only grab the first manager from each team?

Reply:
Is there only ever going to be one manager?